lab-mike/src/main.js

```diff
@@ -0,0 +1,109 @@
+import './style/main.scss';
+import React from 'react';
+import ReactDom from 'react-dom';
+import superagent from 'superagent';
+
+const API_URL = 'http://www.reddit.com/r';
+
+class SearchForm extends React.Component {
```

Comment:
I don't think we can do any destructuring in line 2? We could destructure if we were calling a property off of react, but we are importing the whole react library with a different name yeah?